Adding Tabs to account for OS differences with multiline commands + misc. changes (#42)
osu-tournament-rating/otr-docs: docs/topics/Disaster-Recovery.md

Reviewer: Let's delete this in this PR as well:

```
> Windows users may need to delete the `\` characters before executing commands.
> 
> {style="note"}
```

Author: Deleted in latest commit

> ### Backup the database

Reviewer: This section probably should probably be moved outside of a disaster recovery topic. Don't worry about it for this PR, just making a note.

<tabs group="os">
    <tab id="Windows-backup" title="Windows" group-key="Windows">
        Backup the database into a zip file.<br/>
        <code-block>
        docker exec [container] pg_dump `
        -c `
        -U postgres `
        -d postgres | gzip > /my/dir/dump.gz
        </code-block> 

>         <!-- This command requires a way to execute gzip in Windows, either a software package or 

Reviewer: I would add this in a note block instead of a comment, gzip should be present on the host machine. Alternatively, windows users can use the linux/macOS instructions if executing through the git bash utility.

Author: Ok, I left it as a commend in the code cuz I was unsure what to do. Will do a note in the Windows tab for the gzip/gunzip

            an alternative command prompt to Windows PowerShell-->
    </tab>
    <tab id="Else-backup" title="Linux &amp; macOS" group-key="Else">

>         Backup the database into a zip file.<br/>

Reviewer: ```suggestion
        Backup the database into a compressed archive.<br/>
```

### Backup the database

<tabs group="os">
    <tab id="Windows-backup" title="Windows" group-key="Windows">
        Backup the database into a zip file.<br/>
        <code-block>
        docker exec [container] pg_dump `
        -c `
        -U postgres `
        -d postgres | gzip > /my/dir/dump.gz
        </code-block> 
        <tip>
            gzip needs to be present in the host machine. Alternatively, Windows users can use the Linux/macOS command instructions if executing through the git-bash utility.
        </tip>
    </tab>
    <tab id="Else-backup" title="Linux &amp; macOS" group-key="Else">

>         Backup the database into a compressed archive.<br/>

Reviewer: Delete line 21 and move this line outside of the `tabs` element. Remove the `<br/>` when you do that too.

                docker exec \
                -it [container] psql \
                -U postgres \
                -c "DROP SCHEMA public CASCADE;" \
                -d postgres
                </code-block><br/></li>
            <li>Create the <code>public</code> schema:<br/>
                <code-block>
                docker exec \
                > -it [container] psql \
                > -U postgres \
                > -c "CREATE SCHEMA public;" \
                > -d postgres 
                </code-block><br/></li>
        </ol>
    </tab>
</tabs>

##### Overwrite the database 

<tabs group="os">
    <tab id="Windows-overwrite" title="Windows" group-key="Windows">
        Overwrite your database with the dump:<br/>
        <code-block>
        gunzip `
        -c /my/dir/dump.gz | docker exec `
        -i [container] psql `
        -U postgres `
        -d postgres
        </code-block>
        <tip>
            gunzip needs to be present in the host machine. Alternatively, Windows users can use the Linux/macOS command instructions if executing through the git-bash utility.
        </tip>
    </tab>
    <tab id="Else-overwrite" title="Linux &amp; macOS" group-key="Else">

>         Overwrite your database with the dump:<br/>

Reviewer: Move this outside the tabs section, delete line 98. Remove the <br/> afterward.

> > Cleaning the database is strongly recommended before overwriting it.

Reviewer: Honestly let's make this an affirmative requirement before importing. I always clean before reimporting and failing to do so would probably result in broken imports. Bonus points if you convert all of these steps into a `procedure`, though I'm not sure whether Writerside will even let us do that.

>
{style="note"}

> ##### Cleaning the database

Reviewer: ```suggestion
##### Clean the database
```

 <tabs group="os">
    <tab id="Windows-Schema" title="Windows" group-key="Windows">
        <ol>
            <li>Remove the <code>public</code> schema:<br/>
                <code-block>
                docker exec `

>                 -it serene_keller psql `

Reviewer: ```suggestion
                -it [container] psql `
```

                -U postgres `
                -c "DROP SCHEMA public CASCADE;" `
                -d postgres
                </code-block><br/></li>
            <li>Create the <code>public</code> schema:<br/>
                <code-block>
                docker exec `
                > -it [container] psql `
                > -U postgres `
                > -c "CREATE SCHEMA public;" `
                > -d postgres 
                </code-block><br/></li>
        </ol>
    </tab>
    <tab id="Else-Schema" title="Linux &amp; macOS" group-key="Else">
        <ol>
            <li>Remove the <code>public</code> schema:<br/>
                <code-block>
                docker exec \

>                 -it serene_keller psql \

Reviewer: ```suggestion
                -it [container] psql \
```

                -U postgres \
                -c "DROP SCHEMA public CASCADE;" \
                -d postgres
                </code-block><br/></li>
            <li>Create the <code>public</code> schema:<br/>
                <code-block>
                docker exec \
                > -it [container] psql \
                > -U postgres \
                > -c "CREATE SCHEMA public;" \
                > -d postgres 
                </code-block><br/></li>
        </ol>
    </tab>
</tabs>

> ##### Overwriting the database 

Reviewer: ```suggestion
##### Overwrite the database 
```
